.github/actions/prepare-horreum-report/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Prepare Horreum Report
+description: Prepares JSON report before further processing, i.e. creates the JSON with the env data which gets from the configmap.
+
+inputs:
+  project:
+    description: 'The project namespace.'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - id: prepare-report-file
+      name: Create and Prepare Report File
+      shell: bash
+      # language=bash
+      run: |
+        output_file_prefix="result-"
+        cur_date_iso=$(date --iso-8601=seconds)
+        cur_date_iso_compressed=$(date '+%Y%m%d-%H%M%S')
```

Comment:
just a nit pick, the `cur_date_iso` and `cur_date_iso_compressed` would be slightly different, calling it out just in case.